source/scss/_pf-component-toast-notification.scss

```diff
@@ -0,0 +1,40 @@
+//
+// Toast Notifications
+// --------------------------------------------------
+
+.pf-toast {
+  background-color: $color-pf-white;
+  border-color: $color-pf-black-400;
+  box-shadow: 0 2px 6px $pf-toast-notification-box-shadow-color; // check if we can componentize
+}
+
+.pf-toast__icon {
+  font-size: $pf-alert-icon-font-size;
+  margin: calc(-.75rem - 1px) 1rem calc(-.75rem - 1px) calc(-1.25rem - 1px);
+  padding: $pf-spacer-xs $spacer;
+  box-shadow: 2px 0 5px -2px rgba($color-pf-black-1000, 0.2);
+  &:before { color: transparentize(white, 0.26); }
+  .alert-success & { background-color: $brand-success; }
+  .alert-info & { background-color: $state-info-border; }
+  .alert-warning & { background-color: $brand-warning; }
+  .alert-danger & { background-color: $brand-danger; }
+}
+
+.pf-toast__action {
+  margin-left: $spacer;
+}
+
+.pf-toasts-list {
```

Comment:
@cardosogabriel "hate" is a strong word: I hate this name 😄 lol
can it be -container or something better? maybe take some inspiration from a similar bs element?